min.go

```diff
@@ -0,0 +1,10 @@
+package funky
+
+import "golang.org/x/exp/constraints"
+
+func Min[T constraints.Ordered](min, max: T) T {
```

Comment:
Instead of having min and max locked to two values - how about making them variadic?